opencensus/exporters/trace/stackdriver/stackdriver_exporter.h

```diff
@@ -31,12 +33,18 @@ struct StackdriverOptions {
 
   // The RPC deadline to use when exporting to Stackdriver.
   absl::Duration rpc_deadline = absl::Seconds(5);
+
+  // (optional) By default, the exporter connects to Stackdriver using gRPC. If
+  // this stub is non-null, the exporter will use this stub to send gRPC calls
+  // instead. Useful for testing.
```

Comment:
This is great!
In the envoy OC tracer I would like to see envoys [google](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/grpc_service.proto#envoy-api-msg-core-grpcservice-googlegrpc) grpc stub and service endpoint as additional options.